codegen/Cargo.toml

```diff
@@ -6,7 +6,7 @@ edition = "2021"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-anyhow = { version = "1", default-features = false }
```

Comment:
@bitshifter I enabled the (default) `"std"` feature here to get `From<>` implementations for any `Error` into an `anyhow::Error`, but it looks like you handled that everywhere else by `match` with `Ok(x) => x` and `Err(e) => bail!("some context {}")` - what is desired?

Reply:
Fixed in #308 :)